### PR TITLE
Debug login problems

### PR DESCRIPTION
**Pull request by cpfeiffer**

### CODE REVIEW

**Comment by felipeweb** [comment]
@cpfeiffer could you add some test for the case that the email is null?

File: src/main/resources/log4j.xml [hunk 1]
```diff
@@ -77,7 +84,12 @@
         <appender-ref ref="main"/>
     </category>
     
-    <category name="br.com.caelum.vraptor.simplemail.aws">
+    <category name="org.mamute.auth">
+        <priority value="DEBUG"/>
```
Comment (xdarklight) [inline src/main/resources/log4j.xml]: should this be DEBUG by default?

**Comment by xdarklight** [comment]
I like this change in general - I had some login issues myself due to connection problems to our LDAP server.
Those were really nasty to figure out - so :+1: from me for fixing the log-levels.

File: src/main/resources/log4j.xml (hunk 1 above)
Comment (felipeweb) [inline src/main/resources/log4j.xml]: @cpfeiffer  change here to info

File: src/main/java/org/mamute/auth/LDAPApi.java [hunk 2]
```diff
@@ -227,7 +228,7 @@ private void updateAvatarImage(LDAPResource ldap, Entry entry, User user) {
 	private void createUserIfNeeded(LDAPResource ldap, String cn) throws LdapException {
 		Entry ldapUser = ldap.getUser(cn);
 		String email = ldap.getAttribute(ldapUser, emailAttr);
-		User user = users.findByEmail(email);
+		User user = email != null ? users.findByEmail(email) : null;
```
Comment (felipeweb) [inline src/main/java/org/mamute/auth/LDAPApi.java]: Add some unit tests to this case that email is null

**Comment by felipeweb** [comment]
@cpfeiffer  can you add the unit tests that I mentioned up there so we merge the pull request?

**Comment by cpfeiffer** [comment]
Yes, sorry, I will eventually do it, I'm just too busy with other things at the moment. I had to debug login problems at work and needed to fnid out the actual ldap error cause, that's why I quickly updated the patch.
